src/notebooks/execution/cellExecution.node.ts

```diff
@@ -608,6 +624,16 @@ export class CellExecution implements IDisposable {
         output: ExecuteResult | DisplayData | nbformat.IStream | nbformat.IError | nbformat.IOutput,
         clearState: RefBool
     ) {
+        if (
+            (isCI || this.context.extensionMode === ExtensionMode.Test) &&
+            process.env.IS_WIDGET_TEST &&
+            output.data &&
+            typeof output.data === 'object' &&
+            WIDGET_MIMETYPE in output.data
+        ) {
+            // eslint-disable-next-line @typescript-eslint/no-explicit-any
+            (output.data[WIDGET_MIMETYPE] as any)['_vsc_test_cellIndex'] = this.cell.index;
```

Comment:
Will discuss in standup, Will need this and or https://github.com/microsoft/vscode-jupyter/issues/9629
For now going with something to get the tests working (this only applies to CI scenario).

Basically we need a way to map outputs in webview to outputs of a cell in extension host.

Reply:
Was discussed in standup, going with temp solution until #9629 is implemented. Completing #9629 requires a bit more work and I'd rather do that in a separate PR (as that will require upstream changes as well)

Reply:
Why not just put this there all the time? Seems like it doesn't matter much if this is always there?

That way there won't be all of this extra checking here (which is going to have to change for web support)

Reply:
this will get stored in the json(ipynb), which isn't correct

Reply:
Isn't correct but shouldn't hurt anything. Like who cares?

Reply:
Agreed, however I think since this is not absolutely necessary, we can leave this just for tests.
I.e. adding for production isn't going to buy us anything.

Reply:
Except this will have to be refactored for web in order to determine if running under tests or not. More complexity when the added data doesn't hurt anything.

Reply:
ill remove `process.env.IS_WIDGET_TEST` and see if the tests works, that way we don't need anything special, except the `context` object.
then you won't need any special cashing for web.
basically i don't want users to end up with a lit of unnecessary scm changes which they have complained about in the past.
agreed, let's not make it too complicated, hence will remove the process and the like and only use `context` object.